packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -52,4 +53,190 @@ void main() {
 
     expect(tester.element(find.byKey(key)).size, equals(const Size(800.0, 60.0)));
   });
+
```

Comment:
Shouldn't we have a test that verifies that the divider (the text underline) appears when it's supposed and that it appears in the correct place?